packages/components/src/components/alert/alert.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { Alert } from './alert';
+
+describe('Alert', () => {
+	let element;
+	beforeEach(async () => {
+		element = new Alert();
+		jest.useFakeTimers();
+	});
+
+	const html = `<t-alert>Notifications</t-alert>`;
```

Comment:
Pleas move `<t-alert>Notifications</t-alert>` to line 17 as it is not reused anywhere else